lib/core/src/sdk.rs

```diff
@@ -625,6 +625,9 @@ impl LiquidSdk {
                         let partial_sync = (is_new_liquid_block || is_new_bitcoin_block).not();
                         _ = cloned.sync(partial_sync).await;
 
+                        #[cfg(all(target_family = "wasm", target_os = "unknown"))]
```

Comment:
NIT: Perhaps it is better to put this line at the end of the code block for clarity?